Light/include/light/rendering/renderer.hpp

```diff
@@ -16,20 +16,25 @@ namespace Light
 		static void init();
 		static void onWindowResize(uint32_t width, uint32_t height);
 
-		static void beginScene(Camera& camera, glm::mat4 camera_view);
+		static void beginScene(Camera& camera, glm::mat4 camera_view, glm::vec3 camPos);
```

Comment:
No need to take position as a separate parameter
Can extract camera position from view matrix